x-pack/plugins/alerting/server/alert/create_alert_factory.ts

```diff
@@ -116,6 +123,22 @@ export function createAlertFactory<
           );
         }
       },
+      trimRecovered: (recoveredAlerts: TrimRecoveredOpts[]) => {
+        let earlyRecoveredAlerts: TrimRecoveredOpts[] = [];
+        if (recoveredAlerts.length > maxAlerts) {
+          recoveredAlerts.sort((a, b) => {
+            return a.flappingHistory.length - b.flappingHistory.length;
+          });
+
+          earlyRecoveredAlerts = recoveredAlerts.splice(maxAlerts * 1);
```

Comment:
what is this `*1` for?

Reply:
No I think I accidentally left that in thanks for catching  :)